public/window/ready.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const fs = require('fs');
+
+function handleCommandLine(mainWindow) {
+  let directory;
+  if (path.isAbsolute(process.argv[1])) {
```

Comment:
As `process.argv[1]` is mentioned three times, I think that it would be more readable to give it a name, before starting the `if` condition and replace all these calls with that variable.

Also, an error should be displayed if there is no given argument, what do you think?
Otherwise I guess it would show a native error.

Reply:
It makes total sense. I'll do that.